style.css

```diff
@@ -505,7 +507,129 @@ textarea:focus {
 
 /* Contact me ends here */
 
+.detail-card {
+  display: none;
+}
+
+.detail-card--show {
```

Comment:
- [ ] You did a great job generating the works section dynamically using javascript. :+1: Let's work on some minor details to match the Figma design. Kindly use the correct background color for the popup. There is also some amount of blur in the background. Please update it accordingly. ✍️

**Your design**
![Screenshot from 2023-06-23 10-00-15](https://github.com/fatima-najafi/Portfolio/assets/17243325/20542332-ff78-48be-b06f-553c32884cef)

**Figma design**
![Screenshot from 2023-06-23 10-00-31](https://github.com/fatima-najafi/Portfolio/assets/17243325/139e0796-285d-4005-8593-c8cd0e8cf3b0)

Reply:
- [ ] Great work improving matching the background color of your pop up window to the Figma. Kindly note that there is a level of transaparency in the Figma background. This transparency is not present in your design. The previous reviewer hinted on it and pointed out that there is a blur in the background. kindly implement this too & compare. 
**Your design**
![pop](https://github.com/fatima-najafi/Portfolio/assets/99656687/243a21d2-2001-4d28-86b1-20549387cbce)

**Figma**
![pop1](https://github.com/fatima-najafi/Portfolio/assets/99656687/9f7acfb9-ac17-4872-b62e-58ff5cec5fb5)

- [OPTIONAL] The pop up window should fit properly on the screen without the need to scroll vertically or horizontally 📓